tests/memex/__init___test.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+
+import memex
+
+
+def test_set_groupfinder(pyramid_config):
```

Comment:
Sure, the reason why I did that is that I'm a bit nervous because of memex' default-permissive behaviour when the groupfinder isn't set. But I'm just writing a functional test for that and then I will remove these.